### PR TITLE
[DocsArchive-1.0.21283.5](2021-12-10-14-38-57)Auto archive content

### PR DESCRIPTION
**Pull request by IEvangelist**
***Please don't merge this PR until target archive repo PR https://github.com/dotnet/docs-archive-pr/pull/27 is merged into live branch.*** 
 Auto archive content to https://github.com/IEvangelist/docs-archive-pr.git

### Details

We're archiving the _Framework Design Guidelines_ content until we work with the authors and publishers to acquire permission to the latest version.

The latest content is available here, [Framework Design Guidelines: Conventions, Idioms, and Patterns for Reusable .NET Libraries](https://aka.ms/dotnet/framework-design-guidelines)

### CODE REVIEW

**Comment by BillWagner** [comment]
@IEvangelist Do you want to mark this as "draft" while we work through the current discussion?

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -1,5 +1,245 @@
 {
     "redirections": [
+    {
+      "source_path": "docs/standard/design-guidelines/abstract-class.md",
```
Comment (Youssef1313) [inline .openpublishing.redirection.json:4]: These need to go to `.openpublishing.redirection.standard.json`

**Comment by IEvangelist** [comment]
Abandoning this for now. We still do not have an agreement in place, but issues have at least been less frequent. We may need to revisit this at a later date.

**Comment by SteveDunn** [comment]
Is this PR to archive the collections guidance on collections? I can still see that it live, and I'm still getting team members using this guidance.

**Pull request closed without merging**